### PR TITLE
Add `session_id` field to `AskedQuestion` entity; rename some enities

### PR DESCRIPTION
**Pull request by hromogen**
## PR Checklist

PR meets the following requirements:

## PR Type

PR introduces the following changes:

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Other... Please describe: DB related changes
```

## Actual/New behavior

AskedQuestion entity now has reference to the current session (via ID). Also some renaming performed

Issue Number: #43

### CODE REVIEW

File: server/src/entity/AskedQuestion.ts [hunk 1]
```diff
@@ -17,6 +17,12 @@ export class AskedQuestion {
   })
   questionId: string;
 
+  @Column('uuid', {
+    name: 'session_id',
```
Comment (andrIvash) [inline server/src/entity/AskedQuestion.ts]: If we are talking about competition then this should be contest_id. A session is linked to the user registration process.

File: server/src/migration/1553647346431-QuestionMigration.ts [hunk 2]
```diff
@@ -1,45 +1,45 @@
-import {MigrationInterface, QueryRunner, Table} from 'typeorm';
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
 
 export class Question1553647346431 implements MigrationInterface {
+  async up(queryRunner: QueryRunner): Promise<any> {
```
Comment (kovnick) [inline server/src/migration/1553647346431-QuestionMigration.ts]: You shouldn't edit existing migrations. Create a new one instead.

File: server/src/entity/Question.ts [hunk 3]
```diff
@@ -1,36 +1,36 @@
 import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
 
-@Entity({name: 'qms_question'})
+@Entity({ name: 'questions' })
```
Comment (kovnick) [inline server/src/entity/Question.ts]: @alexZogich working on Questions model/service now. Sync with him.

**Comment by hromogen** [comment]
As some of the changes introduced here are covered in another PR will add remaining (after improvement) in another PR (probably covering session layer back-end)

**Pull request closed without merging**